src/libs/ReportActionsUtils.ts

```diff
@@ -148,6 +148,10 @@ function isCreatedAction(reportAction: OnyxInputOrEntry<ReportAction>): boolean
 function isDeletedAction(reportAction: OnyxInputOrEntry<ReportAction | OptimisticIOUReportAction>): boolean {
     const message = reportAction?.message ?? [];
 
+    if (reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.POLICY_CHANGE_LOG.INVITE_TO_ROOM) {
```

Comment:
```suggestion
    if (isInviteOrRemovedAction(reportAction)) {
```

Could we use `isInviteOrRemovedAction`?